Tasks/AzureCLI/azureclitask.ts

```diff
@@ -15,7 +15,12 @@ export class azureclitask {
         try {
             tl.setResourcePath(path.join( __dirname, "task.json"));
 
-            var bash = tl.createToolRunner(tl.which("bash", true));
+            var tool;
+            var scriptType: string = tl.getInput("scriptType", true);
```

Comment:
not needed.  if script location supplied, determine based on extension and doc that.  Don't make them supply the script and tell you what type it is.